common/components/WalletDecrypt/components/PrivateKey.tsx

```diff
@@ -17,7 +18,7 @@ interface Validated {
 }
 
 function validatePkeyAndPass(pkey: string, pass: string): Validated {
-  const fixedPkey = stripHexPrefix(pkey);
+  const fixedPkey = stripHexPrefix(pkey).trim();
```

Comment:
perhaps the `trim` should happen in `stripHexPrefix`? A minor opinionated nitpick.